internal/testutil/assert/difflib.go

```diff
@@ -1,19 +1,8 @@
-// Package difflib is a partial port of Python difflib module.
-//
-// It provides tools to compare sequences of strings and generate textual diffs.
-//
-// The following class and functions have been ported:
-//
-// - SequenceMatcher
-//
-// - unified_diff
-//
-// - context_diff
-//
-// Getting unified diffs was the main goal of the port. Keep in mind this code
-// is mostly suitable to output text differences in a human friendly way, there
-// are no guarantees generated diffs are consumable by patch(1).
-package difflib
+// Copied from https://github.com/pmezard/go-difflib/blob/master/difflib/difflib.go
```

Comment:
`go-difflib` is a very small, unmaintained [library](https://github.com/pmezard/go-difflib). Instead of adding it as a full dependency for the new copy of `testify`, I opted to copy `difflib.go` into `internal/assert`. It seems we need _most_ of the functions in this file, but maybe not all. I'm looking into reducing this file further.

Reply:
Done.

> P.S. Maybe other people would benefit from a modernized fork of go-difflib. Are there any existing forks? Should we create a MongoDB fork of go-difflib that supports modules?

I've asked whether other Go users use go-difflib in the golang slack channel.

Reply:
There is actually already a MongoDB-maintained and modernized fork of `go-difflib` at https://github.com/mongodb-forks/go-difflib. It's difficult to find that fork by searching, tho, so I suspect it doesn't get much use.